gnd/src/main/res/layout/fragment_terms.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+
```

Comment:
could you also attach images / links to screenshots to allow us to visually see this.

Reply:
ok

Reply:
let me know when you have this ready. Everything else looks good.

Reply:
![Screenshot_20210325-193017](https://user-images.githubusercontent.com/12099245/112485246-bb349000-8da0-11eb-9d9e-9b48d88a32ca.jpg)